test/extensions/common/wasm/test_data/test_cpp.cc

```diff
@@ -213,6 +214,15 @@ WASM_EXPORT(uint32_t, proxy_on_vm_start, (uint32_t context_id, uint32_t configur
     if (pathenv != nullptr) {
       FAIL_NOW("PATH environment variable should not be available");
     }
+    // Call to clock_time_get on monotonic clock should be available.
+    std::chrono::steady_clock::time_point t1 = std::chrono::steady_clock::now();
+    for (auto i = 0; i < 10; i++) {
+      // Busy wait since "sleep" not available yet.
+    }
```

Comment:
I'd be surprised if this isn't optimized out. Perhaps move `t1` to the beginning of this `if` block? All the other WASI calls should move the clock forward.

Reply:
yeah good point. Thanks!